test/contracts/apps/app_proxy.js

```diff
@@ -16,9 +14,11 @@ const ERCProxyMock = artifacts.require('ERCProxyMock')
 const KernelSetAppMock = artifacts.require('KernelSetAppMock')
 
 const APP_ID = hash('stub.aragonpm.test')
-const ZERO_ADDR = '0x0000000000000000000000000000000000000000'
 const EMPTY_BYTES = '0x'
```

Comment:
Wonder if we can replace this constant for https://github.com/aragon/contract-helpers/blob/master/packages/test-helpers/src/bytes.js#L1

Reply:
All the tests using `0x00` as empty bytes were failing, we should change it in the helpers I think